tests/Tests/Unit/ApiExceptionTest.php

```diff
@@ -203,10 +204,105 @@ public function testCreateFromApiResponse($metadata, $metadataArray) {
         $this->assertSame($metadata, $apiException->getMetadata());
     }
 
+    public function getRestMetadata()
+    {
+        $unknownBinData = [
+            [
+                '@type' => 'unknown-bin',
+                'data' => '<Unknown Binary Data>'
+            ]
+        ];
+        $asciiData = [
+            [
+                '@type' => 'ascii',
+                'data' => 'ascii-data'
+            ]
+        ];
+        $retryInfoData = [
+            [
+                '@type' => 'google.rpc.retryinfo-bin',
+                'retryDelay' => [
+                    'seconds' => 1,
+                    'nanos' => 2,
+                ],
+            ]
+        ];
+        $allKnownTypesData = [
+            [
+                '@type' => 'google.rpc.retryinfo-bin',
+            ],
+            [
+                '@type' => 'google.rpc.debuginfo-bin',
+                "stackEntries" => [],
+                "detail" => ""
+            ],
+            [
+                '@type' => 'google.rpc.quotafailure-bin',
+                'violations' => [],
+            ],
+            [
+                '@type' => 'google.rpc.badrequest-bin',
+                'fieldViolations' => []
+            ],
+            [
+                '@type' => 'google.rpc.requestinfo-bin',
+                'requestId' => '',
+                'servingData' => '',
+            ],
+            [
+                '@type' => 'google.rpc.resourceinfo-bin',
+                'resourceType' => '',
+                'resourceName' => '',
+                'owner' => '',
+                'description' => '',
+            ],
+            [
+                '@type' => 'google.rpc.help-bin',
+                'links' => [],
+            ],
+            [
+                '@type' => 'google.rpc.localizedmessage-bin',
+                'locale' => '',
+                'message' => '',
+            ],
+        ];
+
+        return [
+            [
+                [[]], [[]],
+                [$unknownBinData, $unknownBinData],
+                [$asciiData, $asciiData],
+                [$allKnownTypesData, $allKnownTypesData]
```

Comment:
Your test function doesnt accept a second argument so you don't need to supply the same argument twice

Reply:
Oops thank you, done.